pkgs/applications/version-management/git-and-tools/git-remote-hg/default.nix

```
@@ -1,33 +1,32 @@
-{ stdenv, fetchgit, mercurial, makeWrapper,
+{ stdenv, fetchFromGitHub, mercurial, makeWrapper,
   asciidoc, xmlto, docbook_xsl, docbook_xml_dtd_45, libxslt, libxml2
 }:
 
 stdenv.mkDerivation rec {
-  rev = "e716a9e1a9e460a45663694ba4e9e8894a8452b2";
-  version = "0.2-${rev}";
+  version = "1.0.0";
   name = "git-remote-hg-${version}";
```

```suggestion
  pname = "git-remote-hg";
```

Please reuse the attribute where it makes sense

> stdenv.mkDerivation may now be passed pname and version attributes instead of name, and will now automatically construct the package name by joining them with a dash.